client/src/components/existing-request/existing-request-display.vue

```diff
@@ -26,66 +26,80 @@
       </transition>
 
       <transition mode="out-in" name="fade">
-        <v-row class="mt-5" :key="refreshCount">
-          <!-- labels and values -->
-          <v-col cols="9" class="py-0">
-            <v-row dense>
-              <v-col cols="12">
-                <span>Last Update:</span>
-                &nbsp;{{ lastUpdate }}
-              </v-col>
-              <v-col cols="12">
-                <span>Request Status:</span>
-                &nbsp;{{ requestStatusText }}
-                <v-icon v-if="isAlertState" small color="error">mdi-alert</v-icon>
-              </v-col>
-              <v-col cols="12">
-                <span>Priority Request:</span>
-                &nbsp;{{ isPriorityReq ? 'Yes' : 'No' }}
-              </v-col>
-              <v-col cols="12">
-                <span>Expiry Date:</span>
-                &nbsp;{{ expiryDate }}
-              </v-col>
-              <v-col cols="12">
-                <span>Reapplications Remaining:</span>
-                &nbsp;{{ reapplicationsRemainingText }}
-              </v-col>
-              <v-col cols="12" v-if="nr.consentFlag && (nr.consentFlag !== 'N')">
-                <span>Consent Rec'd:</span>
-                &nbsp;{{ consentDate }}
-              </v-col>
-              <v-col cols="12">
-                <span>Applicant Name:</span>
-                &nbsp;{{ nr.applicants.lastName }},
-                &nbsp;{{ nr.applicants.firstName }}
-              </v-col>
-              <v-col cols="12">
-                <span>Address:</span>
-                &nbsp;{{ address }}
-              </v-col>
-            </v-row>
-          </v-col>
+        <v-container class="nr-data pa-0">
```

Comment:
If you select the option to hide whitespace (gear icon at top of Files Changed page), this doesn't look so bad.